oci/private/image.bzl

```diff
@@ -174,6 +174,14 @@ def _oci_image_impl(ctx):
         inputs = depset(transitive = inputs_depsets),
         arguments = [args],
         outputs = [output],
+        # Don't convert arguments like --entrypoint=/some/bin to --entrypoint=C:/msys64/some/bin
```

Comment:
nit: we should only set this on windows?

Reply:
sure, I'll make it conditional

Reply:
actually I think I could move it into the .bat wrapper, will test that

Reply:
that'd be awesome actually.

Reply:
instead of being part of action cache key, it can be a runtime detail.

Reply:
I failed to get that to work, it messes with something and causes

`ERROR: C:/users/alex/documents/github/rules_oci/e2e/smoke/BUILD.bazel:28:12: OCI Tarball //:tar failed: (Exit 1): wrap_tar.bat failed:tar.exe: could not chdir to '/tmp/tmp.Ekhtep7ycr'`